pkg/restore/restore.go

```diff
@@ -1655,22 +1656,39 @@ func (ctx *restoreContext) getOrderedResourceCollection(backupResources map[stri
 // getSelectedRestoreableItems applies Kubernetes selectors on individual items
 // of each resource type to create a list of items which will be actually
 // restored.
-func (ctx *restoreContext) getSelectedRestoreableItems(resource, targetNamespace, originalNamespace string, items []string) (restoreableResource, Result, Result) {
-	var res restoreableResource
+func (ctx *restoreContext) getSelectedRestoreableItems(resrc, targetNamespace, originalNamespace string, items []string) (restoreableResource, Result, Result) {
```

Comment:
I'm wondering if it would be better to go further and do `s/resrc/groupResource` because that string does include the group. What do you think @carlisia?

Reply:
Do you think `groupResource` could signal that it is specifically only group resources? If not, go ahead and change it. I'll approve whatever you decide.